-Password parameter needs secure-string

karthikeyanVK opened the pull request:
-Password needs secure-string or we should use -p. Please find the error which we got when using -Password.

New-AzureRmADServicePrincipal : Cannot bind parameter 'Password'. Cannot convert the "xxxxxxxxxxxxxxxxxxxxxxxxyyyyyyyyyyyyyyy" value of type "System.String" to type "System.Security.SecureString".

azurecla commented:
Hi __@adhurwit__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

BryanLa commented:
@cabailey - can you please review when you have a minute?

cabailey commented:
Hi @adhurwit, Thanks for suggesting this extra link, although I'm not able to find the referenced article for encrypting and decrypting blogs using Key Vault, so we wouldn't be able to accept this pull request at this time. But in addition, we've reached our max number of links I want to include in this Next Steps section. Too many, and they can quickly become overwhelming for first-time customers, which is the target audience for this article.

BryanLa commented:
@adhurwit @cabailey I'll close the PR for now, until the other article is done so the link would be valid.  Also, @adhurwit given that you are more than a casual contributor, it would be best for you to work against our private repo.  You will get quicker response, and we also have bot automation that provides article validation and CI for quick preview of the articles.. You can learn more in the Contributors' Guide: https://github.com/Azure/azure-content/blob/master/contributor-guide/contributor-guide-index.md.  Just be sure to take note of steps that are demarcated with "If you are a MSFT employee..."

karthikeyanVK committed:
-Password parameter needs secure-string 

-Password needs secure-string or we should use -p. Please find the error which we got when using -Password.

New-AzureRmADServicePrincipal : Cannot bind parameter 'Password'. Cannot convert the "xxxxxxxxxxxxxxxxxxxxxxxxyyyyyyyyyyyyyyy" value of type "System.String" to type "System.Security.SecureString".
